pandas/tests/arithmetic/test_timedelta64.py

```diff
@@ -1987,6 +1987,20 @@ def test_td64arr_div_numeric_scalar(self, box_with_array, two):
         with pytest.raises(TypeError, match="Cannot divide"):
             two / tdser
 
+    @pytest.mark.parametrize("two", [2, 2.0, np.array(2), np.array(2.0)])
+    def test_td64arr_floordiv_numeric_scalar(self, box_with_array, two):
```

Comment:
We have versions above of this test for multiplication and division, so adding one for floordiv as well